File: crear_datasets.R
library(tidyverse)
library(feather)
library(purrr)

n_categorias = 1000
n_registros = 10^7

categorias = map_chr(1:n_categorias,~paste(sample(letters,5,replace = T),collapse = ""))

dataset = data.frame(
  categoria = sample(categorias,n_registros,replace = T),
  valor = runif(n_registros)
)

write_feather(dataset,'dataset_test.feater')

File: crear_datasets.jl
using Queryverse
using Feather
using DataFrames
using Random

n_categorias = 1000
n_registros = 10^7

categorias = [randstring(5) for x in 1:n_categorias]

dataset = DataFrame(
    categoria = rand(categorias,n_registros),
    valor = rand(n_registros)
)

#save("dataset_test.feater",datos)
Feather.write("dataset_test_jl.feather",dataset)
# save("dataset_test_jl.feater",dataset)
